### ISSUE TITLE
Idea to improve percentage of positive test reports

### ISSUE THREAD

**Issue by christina002** [corona-warn-app/cwa-wishlist#282]
## Current Implementation

The current interface for registering tests and reporting results is rather confusing for most users, especially given the fact that a lot of labs are not even connected to this app.

To give you a real-world example (my own son tested positive a few weeks ago):

- Son goes to a test station
- Son receives an email with a QR code to register the test, ignored for time time being
- Son receives an email with the positive test result, goes into quarantine as instructed (and confirmed by a call of the "Gesundheitsamt")
- Son wants to register his test results and scans the QR code that only registers the test as such, not the positive result
- The lab was not attached to cwa, thus the positive test result never reached the app.

He does not even attempt to enter the TAN coming with the positive test result because he thinks he's already submitted the results via the QR code that only identifies the test as such.

## Suggested Enhancement

As long as a substantial number of labs are not even connected to this app, the feature of registering a test (and potentially getting the results back via the app) should be visually separated from the UI used to submit positive test results. I would even go one step further and remove it.

## Expected Benefits

A substantial impovement of the rate of reported COVID-19 infections via the app.

**Comment by heinezen** [comment]
Hello @christina002 ,

Thank you for your suggestions. We have added your feedback to s ticket in our internal Jira (EXPOSUREAPP-3953) where it will be investigated by the developers and forwarded to be discussed by the RKI.

Since your feature request ist very similar to https://github.com/corona-warn-app/cwa-wishlist/issues/270 and https://github.com/corona-warn-app/cwa-wishlist/issues/287, we have decided to group the three issues together in one ticket. I will also close your issue as a duplicate of github.com/corona-warn-app/cwa-wishlist/issues/270 to keep the discussion in a central place. However, your initial post will not be lost as we will link it in the mentioned issue. You are also free to copy your post over as a comment if you want.

We will update you on news from the development team when they become available.

Regards,
CH

----
Corona-Warn-App Open Source Team
